cmd/client/commandv2/logs.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commandv2
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/megaease/easegress/v2/cmd/client/general"
+	"github.com/spf13/cobra"
+)
+
+// LogsCmd returns logs command.
+func LogsCmd() *cobra.Command {
+	var n int
+	var follow bool
+	examples := []general.Example{
+		{Desc: "Print the most recent 500 logs by default.", Command: "egctl logs"},
```

Comment:
Why not print all logs like other logs by default.

Reply:
Yeah. Here are my concerns.

1. `kubectl logs` will print help information. If print logs `kubectl logs <pod-name>`. And actually `egctl create`, `egctl apply` will print help information. So, based on this, user may think `egctl logs` will print help information. They may think there are commands like `egctl logs std` or `egctl logs http-access` etc. So, by default, I choose `egctl logs` not print all logs, just a few.
2. easegress logs can be really really long. So print all logs may be not necessary... 

Just my naive opinions, I am ok to change it to print all logs.

Reply:
`git log | cat` will output all of them with `cat` to dismantle the pager effect. It just uses a pager to display logs, essentially it prints all of the logs. But `egctl logs | cat` outputs the last 500 lines. Well, I just provided another thought as I don't strongly insist on my point.  You can decide on the design.

Reply:
You are right. I am just a little afraid that in practice, when user run `egctl logs`, and terminal starts to print logs, then they press `CTRL+C`... (It happens to me all the time).
I am more prefer current design that provides a user-friendly default setting...

Reply:
Then it is.